README.md

```diff
@@ -86,9 +81,3 @@ Licensed under either of
 * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
 
 at your option.
-
-<!-- Badges -->
-[travis-url]: https://travis-ci.org/input-output-hk/jorup
-[travis-badge]: https://travis-ci.org/input-output-hk/jorup.svg?branch=master
-[appveyor-url]: https://ci.appveyor.com/project/NicolasDP/jorup
-[appveyor-badge]: https://ci.appveyor.com/api/projects/status/github/input-output-hk/jorup?svg=true
```

Comment:
can you generate a badge for the github action ?

Reply:
done